src/nwb_datajoint/cli/cli.py

```diff
@@ -0,0 +1,292 @@
+from typing import Union
+import click
+from pyrsistent import optional
+import yaml
+
+@click.group(help="Spyglass command-line client")
+def cli():
+    pass
+
+@click.command(help="Insert a session from a .nwb file that lives in the $NWB_DATAJOINT_BASE_DIR directory")
+@click.argument('nwb_file_name')
+def insert_session(nwb_file_name: str):
+    import nwb_datajoint as nd
+    nd.insert_sessions(nwb_file_name)
+
+@click.command(help="List all sessions")
+def list_sessions():
+    import nwb_datajoint.common as ndc
+    a = ndc.Session & {}
+    print(a)
+
+@click.command(help="Insert a new lab team")
+@click.argument('yaml_file_name', required=False)
+def insert_lab_team(yaml_file_name: Union[str, None]):
+    if yaml_file_name is None:
+        print('You must specify a yaml file. Sample content:')
+        print('==========================================')
+        a = yaml.safe_dump({
+            'team_name': 'your_team_name',
+            'team_description': 'optional_team_description'
+        })
+        print(a)
+        return
+
+    import nwb_datajoint.common as ndc
+    with open(yaml_file_name, 'r') as f:
+        labteam_dict = yaml.safe_load(f)
+    ndc.LabTeam.insert1(labteam_dict)
+
+@click.command(help="List all lab teams")
+def list_lab_teams():
+    import nwb_datajoint.common as ndc
+    a = ndc.LabTeam & {}
+    print(a)
+
+@click.command(help="Insert a new lab member")
+@click.argument('yaml_file_name', required=False)
+def insert_lab_member(yaml_file_name: Union[str, None]):
+    if yaml_file_name is None:
+        print('You must specify a yaml file. Sample content:')
+        print('Note that lab_member_name is the primary key')
+        print('==========================================')
+        a = yaml.safe_dump({
+            'lab_member_name': 'first last',
+            'first_name': 'firstname',
+            'last_name': 'lastname'
+        }, sort_keys=False)
+        print(a)
+        return
+
+    import nwb_datajoint.common as ndc
+    with open(yaml_file_name, 'r') as f:
+        x = yaml.safe_load(f)
+    ndc.LabMember.insert1(x)
+
+@click.command(help="List all lab members")
+def list_lab_members():
+    import nwb_datajoint.common as ndc
+    a = ndc.LabMember & {}
+    print(a)
+
+@click.command(help="Insert a new lab team member")
+@click.argument('yaml_file_name', required=False)
+def insert_lab_team_member(yaml_file_name: Union[str, None]):
+    if yaml_file_name is None:
+        print('You must specify a yaml file. Sample content:')
+        print('==========================================')
+        a = yaml.safe_dump({
```

Comment:
Would prefer all single letter variable names get changed to something more descriptive.